client/src/controllers/scores/scores_controller.js

```diff
@@ -65,9 +86,26 @@ class ScoresController {
 		})
 	}
 
+	errorScores(scores) {
+		scores = scores || this._scores || []
+		let duplicateErrors = this.duplicateScores(scores)
+
+		let self = this
+
+
+		let otherErrors = scores.filter(score =>
+			typeof score.teamNumber != "number" || score.round == null ||
+			(!this._loading && this._teamNumberList.indexOf(score.teamNumber) === -1)
+		)
+		let badScores = duplicateErrors.concat(otherErrors)
+		badScores = badScores.filter(function (value, index) { return badScores.indexOf(value) == index })

```

Comment:
Better use an arrow function here

Reply:
For some reason when converting to an arrow function it filters out the entire array. Specifically I changed it to <addr> (value, index) => {badScores.indexOf(value) == index } </addr>.